src/ape/managers/converters.py

```diff
@@ -185,6 +186,20 @@ def convert(self, value: Union[str, datetime, timedelta]) -> int:
             raise ConversionError()
 
 
+class StringDecimalConverter(ConverterAPI):
+    """
+    Convert string-formatted floating point values to `Decimal` type.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        # Matches only string-formatted floats with an optional sign character (+/-),
+        # leading and trailing zeros are optional.
+        return isinstance(value, str) and re.fullmatch("[+-]?[0-9]*[.][0-9]*", value)
```

Comment:
Only nit is if theres a simpler regex to run here that picks up decimal values

Or even just trying to cast to decimal and catching exceptions

Reply:
I've reworked the regex to require digits before and after the decimal, because a "." input was slipping through.

Reply:
using `\d` might even be clearer (lots of alpha [here](https://docs.python.org/3/howto/regex.html))

Reply:
also make sure to use `\.` and not just `.` (which is "any character")

Reply:
That's interesting, I used \d and got a depreciation warning from the re module. I didn't look further into it, but will take another look now.

Reply:
Resolved with ba341bb